lib/influxer/config.rb

```diff
@@ -37,7 +37,7 @@ class Config < Anyway::Config
     def load(*)
       super
       # we want pass @cache value as options to cache store, so we want it to be a Hash
-      self.cache_enabled = false if cache.blank?
+      self.cache_enabled = false if cache.blank? || cache_enabled == false
```

Comment:
Why `cache_enabled = false` if cache is blank? I think the empty cache `{}` should enable the cache. And override method `load`, might not be a good idea =) Because it disables the cache and is not switch to enabled, if you change the configuration after calling load, it may break backward compatibility!

Reply:
i took that actually from your commit (at least i thought so). If it has been enabled by default before it should be left this way. I just wanted to add the possibility to disable the cache using the cache_enabled flag set to false, regardless of the setting of cache property.

when cache.blank? is removed it will probably work as expected before.

Reply:
Sorry, if I misled you. My Pull Request in WIP status, I am push commit and after that I understood it's not a good idea, I'll end the PR when  I'm free.

Reply:
Sorry for being a litte pushy. I just wanted to give a hand on this. just deny the pull request if it is not useful to you.